go/vt/vtgate/planbuilder/testdata/ddl_cases.txt

```diff
@@ -9,7 +9,7 @@
       "Name": "main",
       "Sharded": false
     },
-    "Query": "create table t1(id bigint, primary key(id))"
+    "Query": "create table t1 (\n\tid bigint,\n\tprimary key (id)\n)"
```

Comment:
any reason for added `\n` and `\t` to the format?

Reply:
This is the change in the generated ouput since now we are using the query generated from the parse tree and not the original one, so it is formatted as specifies in the `Format` function for the node.